SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorSettings.h

```diff
@@ -367,6 +367,10 @@ class SPATIALGDKEDITOR_API USpatialGDKEditorSettings : public UObject
 	UPROPERTY(EditAnywhere, config, Category = "Debug", meta = (DisplayName = "Enable spatial debugger in editor"))
 	bool bSpatialDebuggerEditorEnabled;
 
+	/** Override multi worker load balancing strategy and use single worker in the editor. */
+	UPROPERTY(EditAnywhere, config, Category = "Debug", meta = (DisplayName = "Disable multi-worker in editor"))
+	bool bDisableMultiWorker;
```

Comment:
I think this should be `bEnableMultiWorker`, defaults to true, and is set to false when disabling mw.

Reply:
@yumnuska interested on your perspective here

Reply:
I agree, mainly because I generally prefer affirmative names for booleans in general as otherwise they can lead to double negative logic.

Reply:
Can't deal with no double negatives

Reply:
Renamed to bEnableMultiWorker.